core/lib/spree/app_configuration.rb

```diff
@@ -140,6 +146,14 @@ class AppConfiguration < Preferences::Configuration
     #   @return [Integer] Customer returns to show per-page in the admin (default: +15+)
     preference :customer_returns_per_page, :integer, default: 15
 
+    # @!attribute [rw] default_admin_light_theme
+    #   @return [String] Default admin light theme name
+    preference :default_admin_light_theme, :string, default: :classic
+
+    # @!attribute [rw] default_admin_dark_theme
+    #   @return [String] Default admin dark theme name
+    preference :default_admin_dark_theme, :string, default: :classic
```

Comment:
The dark theme is no more in this PR, right?

Reply:
What if it's `nil` to better communicate it's not supported yet?

Reply:
@waiting-for-dev we still need to set a theme for when the system is in dark mode, and by default (like it is now) we'll use the same light theme for when the system is either in dark or light mode. Hope this makes sense.